MSYS2 doesn't use ldconfig.  Most of the patching is currently done in
0000-pacman-msysize.patch, these are just some new occurrences.  Ideally this
removal of ldconfig will be a completely separate patch in the future.

--- pacman-5.2.1/test/pacman/meson.build.orig	2019-08-12 03:26:54.000000000 +0200
+++ pacman-5.2.1/test/pacman/meson.build	2020-05-16 17:14:19.986575000 +0200
@@ -79,9 +79,6 @@
   'tests/ignore006.py',
   'tests/ignore007.py',
   'tests/ignore008.py',
-  'tests/ldconfig001.py',
-  'tests/ldconfig002.py',
-  'tests/ldconfig003.py',
   'tests/mode001.py',
   'tests/mode002.py',
   'tests/mode003.py',
@@ -336,7 +333,6 @@
     join_paths(meson.current_source_dir(), 'pactest.py'),
     '--scriptlet-shell', get_option('scriptlet-shell'),
     '--bindir', meson.build_root(),
-    '--ldconfig', LDCONFIG,
     '--verbose',
     join_paths(meson.current_source_dir(), input)
   ]
